third_party/proton/test/test_profile.py

```diff
@@ -75,6 +75,7 @@ def foo(x, y):
     assert data[0]["children"][1]["frame"]["name"] == "test2"
 
 
+@pytest.mark.skipif(is_hip(), reason="Currently broken after updating to ROCm 7")
```

Comment:
It's unfortunate that many proton tests are broken now on hip

Reply:
Agreed. I'll get somebody to help addressing these broken tests.